instance/tests/models/test_utils.py

```diff
@@ -486,3 +486,35 @@ class DjangoResourceTest(SimpleResourceTestCase):
     Run the same tests as in SimpleResourceTestCase, but using DjangoResource.
     """
     make_resource = DjangoResource
+
+    def setUp(self):
+        self.make_resource.save = Mock()
```

Comment:
This will permanently alter the `save` method of `DjangoResource` and could interfere with other tests. The correct way to mock this in the `setUp` method is to use `patch()`, since it will undo the change when the test ends:

```python
def setUp(self):
    super().setUp()
    patcher = patch.object(self.make_resource, 'save')
    patcher.start()
    self.addCleanup(patcher.stop)
```